lib/android.js

```diff
@@ -211,8 +211,8 @@ function _getApi () {
               getInstances(instance, scope, hClass, useIsAssignableFrom, maxCount, instances);
             };
           },
-
           _ZN3art12StackVisitorC2EPNS_6ThreadEPNS_7ContextENS0_13StackWalkKindEmb: ['art::StackVisitor::StackVisitor', 'void', ['pointer', 'pointer', 'pointer', 'uint', 'pointer', 'bool']],
+          _ZN3art12StackVisitorC2EPNS_6ThreadEPNS_7ContextENS0_13StackWalkKindEjb: ['art::StackVisitor::StackVisitor', 'void', ['pointer', 'pointer', 'pointer', 'uint', 'uint', 'bool']],
```

Comment:
I suppose we should move this line above the other so these two are alphabetically sorted relative to each other.